lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +204,39 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
+        self._seed_urls = config.get_seed_urls()
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        all_links_bs = article_bs.find_all('a')
+        for links_bs in all_links_bs:
+            link = links_bs.get('href')
+        # link = article_bs.get('href')
```

Comment:
don't forget to remove the unused code before handing in

Reply:
done